### ISSUE TITLE
Tip UX could be improved

### ISSUE THREAD

**Issue by oliversalzburg** [microsoft/vscode#73814]
Issue Type: <b>Bug</b>

When you hover over any element in your code, a tip pops up:

![image](https://user-images.githubusercontent.com/1658949/57849893-d6b1d580-77dc-11e9-871f-2b8f11bff5e8.png)

Due to the way it is placed in the UI, it can easily capture clicks, when you actually want to place your cursor in your code, above the line it activated on. Why not allow me to click through it? Instead, the tip captures focus.

For some extra fun, double click it. Then you can drag it around as if it was a document tab...

VS Code version: Code 1.33.1 (51b0b28134d51361cf996d2f0a1c698247aeabd8, 2019-04-11T08:27:14.102Z)
OS version: Windows_NT x64 10.0.17763

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6850K CPU @ 3.60GHz (12 x 3672)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: enabled<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|63.91GB (33.03GB free)|
|Process Argv|landslide/landslide.code-workspace|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (14)</summary>

Extension|Author (truncated)|Version
---|---|---
pascal-formatter|ale|2.2.0
bracket-pair-colorizer|Coe|1.0.61
vscode-eslint|dba|1.9.0
gitlens|eam|9.7.4
EditorConfig|Edi|0.13.0
generateuuid|Fir|0.0.5
JSDocTagComplete|Hoo|0.0.2
vscode-esformatter|how|1.6.0
output-colorizer|IBM|0.1.2
terraform|mau|1.3.11
code-beautifier|mic|2.2.1
vscode-lebab|mrm|2.6.0
vscode-puglint|mrm|2.3.0
vscode-paste-image|mus|1.0.4

(1 theme extensions excluded)

</details>

**Comment by sandy081** [comment]
@oliversalzburg Clicking on the hover should be ignored and instead the cursor should be placed in the line above?

**Comment by oliversalzburg** [comment]
@sandy081 Yeah, something like that. I imagined it like, when you click on the hover, the click position is remembered, the hover is closed, the click position on the hover is translated to the offset the mouse cursor has relative to the editor and a click event is generated at that position instead.

For the time being, I increased the delay before the hover is shown, but now I'm often waiting for it to appear and it feels slow.

What I definitely never want, is to interact with content in the hover. Although I can see that as a use case as well. Maybe some users regularly select text from the hover for further actions.

What definitely feels like a bug, is the drag & drop behavior of the hover after double-clicking it. That's really weird.
